recipes-core/images/nilrt-dkms-image.bb

```diff
@@ -10,5 +17,6 @@ require niconsole-image.inc
 require nilrt-xfce.inc
 require nilrt-initramfs.inc
 require include/licenses.inc
+require nilrt-internal.inc
```

Comment:
Could I suggest a name like `nilrt-proprietary.inc` instead? I think that would better communicate its purpose to customers - who are probably familiar with the "proprietary" and "nonfree" feeds in other distros.